src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -49,6 +57,25 @@ public void addSection(Section requestSection) {
         addSectionInExistSections(requestSection);
     }
 
+    public void deleteSection(Station deletedStation) {
```

Comment:
메서드가 10줄을 넘어가지 않도록 작성해보시기 바랍니다. removeInnerSection 메서드도 같이 점검해보아요

Reply:
```suggestion
    public void deleteSectionBy(Station station) {
```

역을 찾아서 삭제하는 건데 변수명만 보면 이미 삭제된 역을 전달하는 걸로 보입니다.